### ISSUE TITLE
Mar 10, 2018 - Next and Previous Tab Shortcuts Disappeared

### ISSUE THREAD

**Issue by gaveitatry** [Lord-Kamina/keybinder#46]
I'm using Mozilla Firefox 56.0.

For a long time now, I have been using ALT + 1 to go to the Previous Tab.  And ALT + 2 to go to the Next Tab.

I was using your Keybinder Add-on to set up these two shortcuts.

Today, after clearing my Browser History on a whim (just my browser history and download history), suddenly all of my Keybinder shortcut setups disappeared and were restored to defaults.  I had to go through and change them again.  And now I can't find the Previous Tab and Next Tab actions anywhere.

I haven't updated the Keybinder Add-on in a long time.  I have all my Add-ons set to never update.  I only update them manually when I feel that it is safe to do so.

I know that CTRL + PGUP and CTRL + PGDN will let me quickly go to Previous and Next Tabs wihtout any stupid popups etc.  But I don't want to use those keys.  I want to use ALT + 1 and ALT + 2.  I am trying to create a macro that will press CTRL + PGUP and PGDN whenever I press ALT + 1 and 2, but it isn't working the way I want.  It isn't instant like it was before and it keeps doing things like showing the hidden menu you get when you press ALT.

I just want these two actions back in my Keybinder Add-on.  How did they even disappear when I never updated the Add-on?

**Comment by gaveitatry** [comment]
I don't know why it worked before and doesn't work now.  Or why I thought it was a Keybinder Add-on setting and now I don't even see it listed in the add-on.  I had to install an extra add-on for those two shortcuts 'Previous Tab' and 'Next Tab'.

**Comment by Lord-Kamina** [comment]
Besides the fact that tdheextension should be entirely non-functional in
modern versions of Firefox... In the last couple which still worked, there
was a setting (in about:config) that interfered with those commands. So, if
that setting is on (which is the Firefox default) the addon doesn't add the
next/previous tab commands. It was something to do with tab previews I
think.

**Comment by Lord-Kamina** [comment]
Also... This addon is abandoned because of the idiocy with WebExtensions
(and I've stopped using Firefox), but if you aren't already you really
should be using the last version I released.

**Comment by gaveitatry** [comment]
Hi.  Thanks for the replies.  I'm using 2.0.5 on Firefox 56.0.  I'm not sure if 2.0.5 is the latest version or not.  It's a struggle figuring stuff out sometimes.  Like when I click the "check for updates" button, nothing happens that I can see.  Anyway, thank you for the reply.  If it is an about:config setting that I needed to make those things appear, then that is probably what happened.  I probably disabled that one day and then forgot about it or how it was done.  Hopefully I can find it again.  Thanks.

**Comment by gaveitatry** [comment]
If there is a way to view old comments/reviews for your add-on, I could probably find someone posting about it.  But they changed the appearance of all the Firefox Add-on pages and I don't know where to find comment history.

**Comment by gaveitatry** [comment]
Aw, I found the Reviews link.  I will browse through old comments and see if I see anything.  Thank you for everything.

**Comment by gaveitatry** [comment]
I found it.  I had to go to about:config and set ctrlTab.previews to false.  Great.  I hope I don't forget in the future.

I'm still using 56.0 so I have no idea how crappy the new Firefox is or not.  They boast that the changes were necessary and that it will improve the speed and performance and other things by 200%.  That sounds nice.  But if a bunch of addons break and can never ever be fixed or recoded, that is of concern since I NEED those addons.  Anyway, is Vivaldi your browser of choice? I read in one of your old posts from 7 months ago that was what you were using.  Or did you switch to something else?  I will look at Vivaldi tomorrow and see what it is about.  I'm just looking for a fast browser with a lot of addons or ways to customize stuff.  Because since the first day I started using Firefox, I have always been looking to do something and I find an addon that does it for me.  So I need a browser that supports stuff like that.

**Comment by Lord-Kamina** [comment]
Yeah, Vivaldi is definitely still rough around the edges but overall it's a
great browser, and things like customizable shortcuts are built-in.

I originally moved to Firefox when they killed Opera after Opera 12. Now
Firefox went sideways and I'm on Vivaldi (by Opera's founder), things going
full circle.

I do recommend the Vivaldi snapshots btw.
